tools/api-markdown-documenter/src/api-item-transforms/TransformApiModel.ts

```diff
@@ -26,19 +26,6 @@ import { createBreadcrumbParagraph, createEntryPointList, wrapInSection } from "
 /**
  * Renders the provided model and its contents to a series of {@link DocumentNode}s.
  *
- * @remarks
```

Comment:
All of this context is covered by property docs in `ApiItemTransformationOptions`.